### PR TITLE
openeo grass addons: use list of bands from t.info

### PR DESCRIPTION
**Pull request by metzm**
* use list of bands as returned by `t.info`
* add some more checks
* format for GRASS 8
* flake8

### CODE REVIEW

File: t.rast.ndvi/t.rast.ndvi.py [hunk 1]
```diff
@@ -129,26 +130,27 @@ def main():
         new_inputs.append("%s.%s@%s" % (strds, red_band, mapset))
         new_inputs.append("%s.%s@%s" % (strds, nir_band, mapset))
 
-        expression=("float(%(instrds)s.%(nir)s@%(mapset)s - %(instrds)s.%(red)s@%(mapset)s) / "
-                    "(%(instrds)s.%(nir)s@%(mapset)s + %(instrds)s.%(red)s@%(mapset)s)" %
+        expression = ("float(%(instrds)s.%(nir)s@%(mapset)s - %(instrds)s.%(red)s@%(mapset)s) / "
```
Comment (anikaweinmann) [inline t.rast.ndvi/t.rast.ndvi.py:133]: Can't NIR and RED both be 0 in any of the datasets used, and thus NaN come out for the NDVI?

**Review by anikaweinmann — APPROVED** [review]
Only one little question. Otherwise it looks very good.

File: t.rast.ndvi/t.rast.ndvi.py (hunk 1 above)
Comment (metzm) [reply t.rast.ndvi/t.rast.ndvi.py:133]: In theory yes, but at least for Landsat and Sentinel-2, red and nir should always be > 0. Metadata for Sentinel-2 read
```
                <SPECIAL_VALUE_TEXT>NODATA</SPECIAL_VALUE_TEXT>
                <SPECIAL_VALUE_INDEX>0</SPECIAL_VALUE_INDEX>
```

**Comment by mmacata** [comment]
Even with improved pep8 compliance! Nice! Please merge.

**Review by mmacata — APPROVED** [review]

**Pull request merged**